bindata/v3.11.0/openshift-apiserver/defaultconfig.yaml

```diff
@@ -43,6 +43,9 @@ auditConfig:
         # generate an audit event in RequestReceived.
         omitStages:
           - "RequestReceived"
+storageConfig:
+  urls:
+    - https://etcd.openshift-etcd.svc:2379
 apiServerArguments:
   minimal-shutdown-duration:
-  - 3s # give SDN some time to converge
\ No newline at end of file
+  - 3s # give SDN some time to converge
```

Comment:
@mfojtik I'm thinking we should bump this up to the 35 we found for kube-apiserver

Reply:
openshift-apiserver works through endpoint IP directly, not via iptables or ELBs. So if our informers in kube-aggregator work, we should be safe here with 3s.